model/labels/labels_common.go

```diff
@@ -114,6 +114,19 @@ func (ls Labels) Map() map[string]string {
 	return m
 }
 
+// FromMaps returns new sorted Labels from the given maps, overriding each other in order.
+func FromMaps(maps ...map[string]string) Labels {
```

Comment:
Please put this in the `rules` package; the Labels interface is already too big.

Reply:
Well, I put it there as `FromMap` is also there and it was kinda like a wrapper for it.
Do you have any preferences for the file I put?

Reply:
How about `rules/manager.go`